pkg/machines/libvirt-dbus.es6

```diff
@@ -517,7 +517,8 @@ LIBVIRT_DBUS_PROVIDER = {
                         props.available = poolInfo[0][3];
 
                         dispatch(updateOrAddStoragePool(Object.assign({}, dumpxmlParams, props)));
-                        dispatch(getStorageVolumes({ connectionName, poolName: dumpxmlParams.name }));
+                        if (props.active)
+                            dispatch(getStorageVolumes({ connectionName, poolName: dumpxmlParams.name }));
```

Comment:
This change seems to be unrelated to this pull request.

Reply:
It's separate commit, check commits.
I tend to put multiple commits into a PR, that sometimes might seem unrelated, but posting a single line PR just for a small fix like this is a bit of overkill since all tests will be triggered.

Reply:
I know that it's separate commit, that's exactly why I'm asking, whether it's a mistake or intentional.

Well, since you've used that as reasoning I cannot resist to point out that in that case the whole process is broken :) you should be able to merge simple fixes without triggering all tests.

Reply:
Theoretically we can trigger parts of the test-suite as well but we don't do that and it works perfect so far. And having some fixes for the same package inside a PR that was meant to mainly do something else is not a great deal - it makes things quicker without making the commit history dirty.